plugins/peermanager/peer-manager.go

```diff
@@ -0,0 +1,84 @@
+package peermanager
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type PeerManagerService struct {
+	client *rpc.Client
+}
+
+type pseudoNodeInfo struct {
+	Enode string
```

Comment:
Let's add JSON hints here. I don't like relying on struct field names for JSON encoding / decoding when we can be explicit.